### PR TITLE
Update preferences. Addresses #818

### PR DESCRIPTION
**Pull request by LanceMcCarthy**
This addition to the preferences documentation shows a data binding scenario use case

### CODE REVIEW

File: docs/essentials/preferences.md [hunk 1]
```diff
@@ -47,6 +47,32 @@ Preferences.Clear();
 
 In addition to these methods each take in an optional `sharedName` that can be used to create additional containers for preference. Read the platform implementation specifics below.
 
+## Data Binding
+
+You can also use a Preference in a view model property for use with XAML data binding. 
+
+#### View model
```
Comment (conceptdev) [inline docs/essentials/preferences.md]: Heading levels should cascade, so this should be H3 ( `###`). also, add a blank line after the heading.

File: docs/essentials/preferences.md [hunk 2]
```diff
@@ -47,6 +47,32 @@ Preferences.Clear();
...
+    set
+    {
+        if (IsMySettingActive == value)
+            return;
+
+        Preferences.Set("IsMySettingActiveKey", value);
+        OnPropertyChanged();
+    }
+}
+
```
Comment (conceptdev) [inline docs/essentials/preferences.md]: remove whitespace here

File: docs/essentials/preferences.md [hunk 3]
```diff
@@ -47,6 +47,32 @@ Preferences.Clear();
...
+```
+#### View
```
Comment (conceptdev) [inline docs/essentials/preferences.md]: add whitespace before and after the heading

**Review by conceptdev — CHANGES_REQUESTED** [review]
I've made a few style comments to match our markdown standards. Will tag someone for an engineering review. Thanks for taking the time to contribute.

**Comment by LanceMcCarthy** [comment]
Thanks @conceptdev, I've made the requested changes.

**Comment by jamesmontemagno** [comment]
Lance! Thanks for the contribution. For the docs we don't want to mix the core API usage with advanced scenarios since the API is general purpose. I do think that this is a better to put it into the Essentials wiki: https://github.com/xamarin/Essentials/wiki/FAQ-%7C-Preferences

**Comment by LanceMcCarthy** [comment]
The Wiki is indeed a perfect place. I'll look for other requested scenarios and try to help build out the Wiki.

**Pull request closed without merging**